src/ml/modelInterfaces.ts

```diff
@@ -1,54 +1,65 @@
 import { buildImgUrl } from '../api/db/models/utils.js';
+import type { LabelSchema } from '../api/db/schemas/shared/index.js';
+import type { ImageSchema } from '../api/db/schemas/Image.js';
 import SM from '@aws-sdk/client-sagemaker-runtime';
 import sharp from 'sharp';
+import { Config } from '../config/config.js';
 
-const _getImage = async (image, config) => {
+const _getImage = async (image: ImageSchema, config: ModelInterfaceParams['config']) => {
   const url = 'http://' + buildImgUrl(image, config);
 
   try {
     const res = await fetch(url);
     const body = await res.arrayBuffer();
-    let imgBuffer = Buffer.from(body, 'binary');
+    let imgBuffer = Buffer.from(body);
```

Comment:
I'm not 100% sure of this change. The typings for the `from` method are as follows:

```ts
from(
    arrayBuffer: WithImplicitCoercion<ArrayBuffer | SharedArrayBuffer>,
    byteOffset?: number,
    length?: number,
): Buffer;
/**
 * Creates a new Buffer using the passed {data}
 * @param data data to create a new Buffer
 */
from(data: Uint8Array | readonly number[]): Buffer;
from(data: WithImplicitCoercion<Uint8Array | readonly number[] | string>): Buffer;
/**
 * Creates a new Buffer containing the given JavaScript string {str}.
 * If provided, the {encoding} parameter identifies the character encoding.
 * If not provided, {encoding} defaults to 'utf8'.
 */
from(
    str:
        | WithImplicitCoercion<string>
        | {
            [Symbol.toPrimitive](hint: "string"): string;
        },
    encoding?: BufferEncoding,
): Buffer;
```

So, to pass in the `BufferEncoding` (e.g `'binary'`), the first argument should be a `str`, not an `ArrayBuffer`.  As such, I removed the `'buffer'` argument to comply with the required types.